docs/dev-env.md

```diff
@@ -89,6 +89,9 @@ in your home directory with the following line:
 set -g default-command "${SHELL}"
 ```
 
+Tmuxinator tests run 2 lightning nodes. One node runs as a gateway to the federation, and the other node represents the rest of the Lightning network. These tests are conducted to simulate the process of sending and receiving transactions via Lightning network to and from Fedimint.
```

Comment:
Can you please find out which kinds of LN nodes are running and in which function instead of reducing specificity? @dpc mentioned he might be wrong.

Reply:
[CLN is the gateway](https://github.com/fedimint/fedimint/blob/5cf843dfe3ec9d3c80e153abf4e6168d3bfe26ee/scripts/build.sh#L65) node and LND is the "other" node

Reply:
Oh thanks a lot @m1sterc001guy

Reply:
Sorry. I was wrong. :D